### ISSUE TITLE
Should "Complex Content Responses" counters reset when play speed changes?

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/molecules-and-light#325]
For several of these responses, the response is dependent on whether or not it is the first of its kind in a sequence of photon/molecule interactions. But current implementation doesn't consider whether play speed changes. So for instance, the following output is possible.

**[User turns on emitter, sim playing normal speed]**
- "Photons on."
- "Stretching"
- "Stretching".

**[User switches to "Slow" speed]**
- "Photon absorbed. Stretching."

The short version is used because the molecule has already been "stretching" several times in a row. However, the first response for "Slow" speed on absorption is
"Photon absorbed. Bonds of molecule stretch back and forth."

Which one should it be in this case? Should the counters for responses reset when play speed changes?

**Comment by terracoda** [comment]
I think a reset on the counter would be quite wonderful, so the learner gets the detailed response at least once.

**Comment by terracoda** [comment]
At least once for each speed.

**Comment by jessegreenberg** [comment]
OK, thanks!

**Comment by jessegreenberg** [comment]
Done in the above commit.
